codegen/core/src/main/java/software/amazon/smithy/python/codegen/DirectedPythonCodegen.java

```diff
@@ -131,8 +131,7 @@ private void generateSchemas(GenerationContext context, Collection<Shape> shapes
         var index = TopologicalIndex.of(context.model());
         Stream.concat(index.getOrderedShapes().stream(), index.getRecursiveShapes().stream())
                 .filter(shapes::contains)
-                .filter(shape -> !shape.isOperationShape() && !shape.isResourceShape()
-                        && !shape.isServiceShape()
+                .filter(shape -> !shape.isResourceShape()
```

Comment:
This is gonna need to be rebased

Reply:
Yep, also need to add cases in `documents.py` to handle the new enum values